index.js

```diff
@@ -319,3 +357,41 @@ app.get('*', (req, res) => {
 		res.status(404).send('404');
 	}
 });
+
+function loadData() {
+	if(!fs.existsSync(file)) {
+		fs.writeFileSync(file, "{}");
+	}
+	const d = fs.readFileSync(file);
+	return JSON.parse(d);
+}
+function saveData() {
+	fs.writeFile(file, JSON.stringify(stats), function (err) {
+		if(err)
+			console.log(err.message);
+		console.log("save")
+	});
+}
+function initScheduler() {
```

Comment:
maybe add the delay as a property so you can more easily change it in the head of the program